app/CocoaPods/CPPodfileInitController.swift

```diff
@@ -23,8 +23,9 @@ public class CPPodfileInitController: NSObject, CPCLITaskDelegate {
     self.projectURL = xcodeprojURL
     
     super.init()
+
     let task = CPCLITask(workingDirectory: xcodeprojURL.URLByDeletingLastPathComponent!.path,
-      command: "init",
+      command: "init \(xcodeprojURL.lastPathComponent!)",
```

Comment:
I think the only case where this will be nil is if the path is ""? which would be a very broken case to begin with